tests/tracking/test_rest_tracking.py

```diff
@@ -2045,6 +2046,36 @@ def _exclude_system_tags(tags: Dict[str, str]):
     assert trace_info == client.get_trace_info(trace_info.request_id)
 
 
+def test_start_and_end_trace_non_string_name(mlflow_client):
+    # OpenTelemetry span can accept non-string name like 1234. However, it is problematic
+    # when we use it as a trace name (which is set from a root span name) and log it to
+    # remote tracking server. Trace name is stored as mlflow.traceName tag and tag value
```

Comment:
is span name the only case that generate non-string tag ?
can we do the conversion in https://github.com/mlflow/mlflow/blob/4b7e339f4c42860ec27d9bcf35efd352b953b53a/mlflow/tracing/processor/mlflow.py#L167 ?

Reply:
No, users can set non-string tag (maybe accidentally)

Reply:
ok. then current approach makes sense